docs/release/handbook.md

```diff
@@ -0,0 +1,292 @@
+* Handling communication for social media content publication
+
+### Docs Lead
+The Docs Lead is responsible for working with the Release Team to coordinate documentation updates for the next Kubeflow release.
+
+**Prerequisites:** Contributions to the [website repo](https://github.com/kubeflow/website)
+
+**Time Commitments:** This role will require some synchronous communication. The bulk of their work is expected to be in the Documentation phase of the release, but they will also be involved throughout the release by keeping track of issues that might require an update to the docs.
+
+**Responsibilities:**
+* Identifying and tracking new issues that require update to the docs
+* Working with contributors to modify existing docs to accurately represent any upcoming changes
+* Reviewing documentation PRs to ensure quality following the website [Style Guide](https://www.kubeflow.org/docs/about/style-guide/)
+* Migrating the old website [version] documentation and updating it with the new release
+
+### Working Group Liaison
+Person responsible for liaising between the release team and the WG. The will
+be the main contact point for the Release Team to figure out the progress of
+the WG as well as required component versions, throughout the release. If a
+Release Team Member is already part of a WG then they could also serve as the
+liaison.
+
+**Prerequisites:** Contributor in the Working Group and acknowledged from the tech leads of that Working Group
+
+**Time Commitments:** This role can be completely asynchronous. Most of their work is expected to be in the Feature Freeze and Documentation phases.
+
+**Responsibilities:**
+- Providing a roadmap and dependency updates to the Community early in the release cycle
+- Communicating with Release Team during the Feature Freeze about the progress of bug fixes
+
+### Distribution Representative
+
+**Prerequisites:** Work at the organization that owns the distribution
+
+**Time Commitments:** This role can be completely asynchronous. Most of their work is expected to be in the Distribution testing phase. Ideally they should be involved when the first RC is cut in order to alert early for potential issues.
+
+**Responsibilities:**
+- Exposing issues that the distribution owner is facing with the new release
+- Syncing with the release team during distro testing phase, we want a nice tight feedback loop there
+
+## Proposed WG Processes
+
+### Releasing
+
+We are moving to a release cadence of **4 months**, in response to our 3 month cycle being tight for the typical set of features we have wanted to release.The release is planned out with 10 weeks of development, with 6 weeks of feature freeze, documentation work, and other processes. At some point there will be a feature freeze, as described below, in the manifests repo. This means that other WGs cannot request the Manifest WG leads to update the repo to change a component’s YAML files, if the update sets a new image with new features.
```

Comment:
I believe we should define how long each release is going to be supported with bug fixes (for example patch versions like `1.3.1`).

Furthermore, we should consider defining each N'th release as a LTS release (with a longer support window), and therefore a longer release timeline (possibly with other requirements to help ensure its not hard to maintain longer term).

Reply:
Very good point! I'll create an issue so that we can start this discussion and have more feedback

Reply:
Created the issue
https://github.com/kubeflow/manifests/issues/1922